src/main/webapp/map.html

```diff
@@ -1,21 +0,0 @@
-<!DOCTYPE html>
-<html>
```

Comment:
I didn't see the new paris.html file? Did you commit it?

Reply:
Just committed now sorry -- I just did add all and commit but for some reason it missed that file. Should be committed now!